Pod/Classes/TSSwiftMarkdownParser.swift

```diff
@@ -100,47 +98,32 @@ public class TSSwiftMarkdownParser: TSBaseParser {
         strongAndEmphasisAttributes = [NSFontAttributeName: strongAndEmphasisFont]
         
         if withDefaultParsing {
-            
-            addNumberedListParsingWithMaxLevel(0, leadFormattingBlock: { (attributedString, range, level) in
-                var listString = ""
-                for _ in (level - 1).stride(to: 0, by: 1) {
-                    listString = "\(listString)\u{00A0}"
-                }
-                let substring = attributedString.attributedSubstringFromRange(range).string.stringByReplacingOccurrencesOfString(" ", withString: "\u{00A0}")
-                listString = "\(listString)\(substring)"
-                attributedString.replaceCharactersInRange(range, withString: listString)
+            addNumberedListParsingWithLeadFormattingBlock({ (attributedString, range, level) in
+                let substring = attributedString.attributedSubstringFromRange(range).string.stringByReplacingOccurrencesOfString(" ", withString: "\(nonBreakingSpaceCharacter)")
+                attributedString.replaceCharactersInRange(range, withString: "\(substring)")
             }) { attributedString, range, level in
```

Comment:
Can you move this inside the parens? Using trailing closure syntax makes it unclear what this closure's purpose is.